bin/setup

```diff
@@ -4,6 +4,7 @@ source ~/.zshrc
 
 echo -e '\033[32mInstalling activesupport\033[0m'
 gem install activesupport
+gem install mdl
```

Comment:
I wanted to push this to #17 (I don't know why puzzled today, maybe because it's monday)